README.md

```diff
@@ -25,14 +25,37 @@ by creating a pull request to this repository.
 [branch]: https://help.github.com/articles/creating-and-deleting-branches-within-your-repository
 [pr]: https://help.github.com/articles/using-pull-requests/
 
+### Commit messages
+
+Git commit messages should follow our convention and be created with the 
+follow structure:
+
+    KEYWORD: (#1) <subject>
+    
+It is **mandatory** to include a keyword. Possible keywords are:
+* CHANGE: If the code makes a breaking change of the public API
+* NEW: If a new (non-breaking) feature is introduced
+* FIX: For bug fixes to the application
+* OPT: For small changes that don't fall into the above categories
+
+It is **mandatory** to refer to a ticket number  
+It is **mandatory** to provide a short description / subject  
+It is *optional* to include an additional body that describes the commit in 
+more details
+
+Example commit message:
+
+    NEW: (#1) Create a basic website with information on D-Nitro
+  
+
 ### Adding new projects
 New projects maintained by the D-Nitro team should be added to the website
 (and also be linked on the KPN Github page).
 
 ### Testing locally
 You can test your changes locally by running a development server::
 
-    $ jekyll serve --livereload
+    $ bundle exec jekyll serve --livereload
```

Comment:
The first time I needed to run `$ bundle install`, and then I could run `$ bundle exec jekyll serve --livereload`

Reply:
I have added instructions for this